example/README.md

```diff
@@ -10,4 +10,46 @@ The output will be a service `aws-es-proxy-service` running in your namespace wh
 
 In your namespace's path in the [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments/) repository, create a directory called `resources` (if you have not created one already) and refer to the contents of [main.tf](main.tf) to define the module properties. Make sure to change placeholder values to what is appropriate and refer to the top-level README file in this repository for extra variables that you can use to further customise your resource.
```

Comment:
I think this; 
```
[main.tf](main.tf)
```

should be;
```
[main.tf](../main.tf)
```

Reply:
@digitalronin , This README is inside the example/ folder and refers to the main.tf in the example folder. This is what users use when copying the module to the environments repo.